Specs/DateTime/English/DateTimeModel.json

```diff
@@ -13215,5 +13215,101 @@
         }
       }
     ]
+  },
+  {
+    "Input": "What happened in late afternoon.",
+    "Context": {
+      "ReferenceDateTime": "2019-07-17T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "late afternoon",
+        "Start": 17,
+        "End": 30,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "TAF",
```

Comment:
Resolutions for such entities need to contain the corresponding MOD attributes.
Are they not being assigned? Or is the test code just not checking them? Both need to be fixed.